runelite-client/src/main/java/net/runelite/client/plugins/pileindicators/PileIndicatorsOverlay.java

```diff
@@ -60,41 +61,29 @@
 	public Dimension render(Graphics2D graphics)
 	{
 		ArrayList<ArrayList<Actor>> stackList = plugin.getStacks();
-		if (config.drawPileHull())
-		{
-			playerIndicatorsService.forEachPlayer((player, color) ->
-			{
-				if (plugin.isPile(player))
-				{
-					Polygon objectClickbox = player.getConvexHull();
 
-					renderPoly(graphics, config.playerPileColor(), objectClickbox);
-				}
-			});
-		}
 		if (stackList != null)
 		{
 			for (ArrayList<Actor> actorArrayList : stackList)
 			{
 				PileType pileType = plugin.getPileType(actorArrayList);
 				Color pileColor = plugin.getColorByPileType(pileType);
 
-				try 
+				try
 				{
 					Actor actorToRender = actorArrayList.get(0); //guaranteed to have at least two players
-					final String text;
-					if (config.numberOnly())
+					final String pileTypeStr = pileType == PileType.PLAYER_PILE ? "PLAYER" : pileType == PileType.NPC_PILE ? "NPC" : pileType == PileType.MIXED_PILE ? "MIXED" : "";
```

Comment:
was this what you were talking about? @f0rmatme

Reply:
ya i think so